irohad/main/irohad.cpp

```diff
@@ -96,6 +96,14 @@ DEFINE_validator(verbosity, validateVerbosity);
 std::promise<void> exit_requested;
 
 int main(int argc, char *argv[]) {
+  if (std::string(IROHA_COMMIT_HASH) == "") {
+    gflags::SetVersionString("Unknown");
```

Comment:
I guess better to write about how to build or what happened, why we can't handle version.